language/move-lang/tests/move_check/parser/byte_string_invalid_unicode.exp

```diff
@@ -0,0 +1,8 @@
+error: 
+
+   ┌── tests/move_check/parser/byte_string_invalid_unicode.move:3:17 ───
+   │
+ 3 │         b"libra \u00ffffff}"
+   │                 ^^ invalid unicode escape: '\u0'
```

Comment:
The loc reported here should include the "0"

Reply:
Done.